### PR TITLE
[bug 1201545] Update mobile instant search to match mockups

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/sumo/static/sumo/less/mobile/main.less [hunk 1]
```diff
@@ -159,6 +159,17 @@ body {
 
     #instant-search-form {
       display: none;
+      position: relative;
+
+      &:after {
+        color: @textGrey;
+        content: '\f002';
```
Comment (mythmon) [inline kitsune/sumo/static/sumo/less/mobile/main.less]: What's this icon?

**Comment by mythmon** [comment]
The close button doesn't work for me. It looks like the search form covers it entirely, blocking the click events.

**Comment by mythmon** [comment]
Oh, and the search icon in the search box looks a bit misaligned. Is that just me?
shot_30939

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
Nice. r+

**Pull request merged**